_rules/meta-refresh-no-delay-no-exception-bisz58.md

```diff
@@ -26,7 +26,7 @@ accessibility_requirements:
     passed: further testing needed
     inapplicable: further testing needed
   wcag20:2.2.1: # Timing Adjustable (A)
-    secondary: True
+    secondary: This success criterion is **less strict** than this rule. This is because this criterion allows redirects longer than 20 hours. This is why some of the failed examples satisfy this success criterion.
```

Comment:
```suggestion
    secondary: This success criterion is **less strict** than this rule. This is because this criterion allows redirects longer than 20 hours. Some of the failed examples satisfy this success criterion.
```